tests/test_glue/test_schema_registry.py

```diff
@@ -0,0 +1,681 @@
+"""Unit tests for glue-supported APIs."""
```

Comment:
Can we separate these per API?

Reply:
the existing folder structure suggests to put all tests related to a sub-service in one file